### DISCUSSION TITLE
How to show the training and validation accuracy as well as loss in the end of process bar??

### DISCUSSION THREAD

**Discussion by JimXu1989** [Lightning-AI/pytorch-lightning#5885] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Thanks for your GREAT work!!!
I want see the accuracy as  well as the loss, when I fit the model, I tried many way to show the accuracy with the process bar as well as it shows in Keras

    def training_step(self, batch, batch_nb):
        x, y = batch
        y_hat = self(x)
        loss = F.cross_entropy(y_hat, y)
        tensorboard_logs = {'train_loss': loss}
        acc = train_accuracy(y_hat.cpu(), y.cpu())
        # self.log('train_loss', loss, on_step=True, on_epoch=True, prog_bar=True, logger=True)
        # self.log('train_acc_step', acc, on_step=True, on_epoch=True, prog_bar=True, logger=True)
        return loss
        # return {'loss': loss, 'acc':acc, 'log': tensorboard_logs}

But it doesn't work

 - OS: Linux
 - Packaging pip
 - Version 1.1.6
Thanks a lot!!!

**Comment by rohitgr7** [comment]
you need something like this? https://github.com/PyTorchLightning/pytorch-lightning-bolts/blob/master/pl_bolts/callbacks/printing.py
